tests/integration-tests/tests/update/test_update.py

```diff
@@ -529,3 +529,159 @@ def _test_max_vcpus(region, stack_name, vcpus):
 def _test_min_vcpus(region, stack_name, vcpus):
     asg_min_size = get_batch_ce_min_size(stack_name, region)
     assert_that(asg_min_size).is_equal_to(vcpus)
+
+
+@pytest.mark.dimensions("us-west-1", "c5.xlarge", "centos7", "sge")
+@pytest.mark.usefixtures("os", "instance")
+def test_update_sit_update_compute_instance(
```

Comment:
These test is very expensive in terms of time cause it needs to create 5 different clusters and also perform an update on all of them. 
On one side we should not aggregate tests that run on independent clusters in the same test function so that we better leverage the test parallelism. On the other side can we drop some of the combinations? Personally I'd test only one combination with disable_hyperthreading=true and one with cfn_scheduler_slots=cores so that we reduce the tests from 5 to 2.